docs/resources/aws_transfer_users.md

```diff
@@ -0,0 +1,77 @@
+---
+title: About the aws_transfer_users Resource
+platform: aws
+---
+
+# aws\_transfer\_users
+
+Use the `aws_transfer_users` InSpec audit resource to test properties of multiple Transfer users.
+
+## Syntax
+
+Ensure that an user exists.
+
+    describe aws_transfer_users do
+      it { should exist }
+    end
+
+## Parameters
+
+For additional information, see the [AWS documentation on Transfer user](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-transfer-user.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| ARNs | The ARNs of the user. |
+| home_directories | The landing directories for users when they log in to the server using the client. |
+| home_directory_types | The landing directory types you want your users' home directory to be when they log into the server. |
+| roles | The ARNs of the IAM roles that controls your users' access to your Amazon S3 bucket or EFS file system. |
+| ssh_public_key_counts | The ssh public key counts of the user. |
```

Comment:
Is this correct? `ssh_public_key_counts`

Reply:
Thanks , Updating this